### PR TITLE
created README with about, setup and account creationg section

### PR DESCRIPTION
**Pull request by n-flint**
# Necessary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
- Created a README.md
- Includes an about section, setup section, as well as documentation on account creation.

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed (Please describe happened)

# Checklist:

- [x] My code has no unused/commented out code
- [] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+## Endpoints
+  - [New Account Creation](#new-account-creation)
+
+
+## New Account Creation
+
+Send a POST request with an email, password and password confirmation. Successful response inludes a newly generated API key for the new user, and the user is saved in the database.
```
Comment (earl-stephens) [inline README.md]: Maybe add that the status code for a successful response as well?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  ```
+  {
+    "email": "my_email@example.com",
+    "password": "password"
+    "password_confirmation": "password"
+  }
+  ```
+  ##### Response
+  ```
+  status: 201
```
Comment (earl-stephens) [inline README.md]: So ignore the earlier comment - you have the status code shown here.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  body:
+
+  {
+    "api_key": "jgn983hy48thw9begh98h4539h4",
+  }
+  ```
+  ##### Requirements
+  - A unique email must be provided
+  - Password and confirmation both must match
```
Comment (earl-stephens) [inline README.md]: If you have the code built out, do you want to say what the response/status code is for a failed call?

File: README.md (hunk 1 above)
Comment (n-flint) [reply README.md]: Nice catch, will do!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+# Expresso Weather
+
+Expresso Weather is a JSON API that uses the Dark Sky API along with the Google Geocoding API to return weather information based on a location provided by the user. The user must include a valid API key. The data returned includes houlry and daily objects base on the current weather. These objects include a summary, icon, temperature, percipitation percentages, as well as wind and visibility information.
```
Comment (RyanDBarnett) [inline README.md]: You need to change houlry to hourly!

Comment (RyanDBarnett) [reply README.md]: I like how you described the objects information.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+# Expresso Weather
+
+Expresso Weather is a JSON API that uses the Dark Sky API along with the Google Geocoding API to return weather information based on a location provided by the user. The user must include a valid API key. The data returned includes houlry and daily objects base on the current weather. These objects include a summary, icon, temperature, percipitation percentages, as well as wind and visibility information. 
+
+## Learning Goals
+  - Utilize a project board to create and track details for project completion.
+  - Practice written technical communication with concise and consistent git commits and clear pull requests.
+  - Implement testing in JavaScript.
```
Comment (RyanDBarnett) [inline README.md]: Implement testing with Jest? Maybe add a reference to Jest.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+## Setup
+1. Clone this repository
+2. Install npm with the command `npm install`
+3. Create a sequelize database `npx sequelize db:create`
+3. Migrate the with database `npx sequelize db:migrate`
+4. Run your npm server `npm start`
+
+  ### Testing
+  1. Globally isntall Jest
```
Comment (RyanDBarnett) [inline README.md]: install

File: README.md (hunk 3 above)
Comment (n-flint) [reply README.md]: I do have that built out. Thanks!

File: README.md (hunk 4 above)
Comment (n-flint) [reply README.md]: Thanks!

File: README.md (hunk 5 above)
Comment (n-flint) [reply README.md]: Thank you, I will make it a link.

**Pull request merged**